### PR TITLE
Mitschriften Systementwicklung 12.05.2021

### PR DESCRIPTION
**Pull request by RvNovae**
Ergänzungen sind gern gesehen.

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- SWE-KASCHE/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- SWE-KASCHE/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWE-KASCHE/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ABWL-LEDER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/ASM-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/DBS-DORENDORF/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/PRO-KASCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/SWS-KUSCHE/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/EMB-GUENTHER/README.md
  - /home/runner/work/dhge-pi19-sem4/dhge-pi19-sem4/OOP-CASSELT/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- SWE-KASCHE/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

**Comment by RvNovae** [comment]
DIe linting Fehler ergeben meines Erachtens eigentlich recht wenig sinn. Das soll eigentlich schon so indentiert sein 🤔

**Comment by ZeroPointMax** [comment]
Fixed it! :D

File: SWE-KASCHE/README.md [hunk 1]
```diff
@@ -268,3 +269,108 @@ $\rightarrow$ Abschätzung der Effizienz
...
+>- Montage
+>- Beschichtung
+>
+> insgesamt 13 verschiedene Programme
+>
+> Frage: Systementwurf um diesen Produkt-Entstehungsprozess zu unterstützen
+>
+> Anzahl an Konvertern bestimmen
+>
+> *???*
```
Comment (nukerxy) [inline SWE-KASCHE/README.md:364]: 👍

**Review by nukerxy — APPROVED** [review]

**Review by ZeroPointMax — APPROVED** [review]

**Pull request merged**